tests/desktop-e2e/src/control.ts

```
@@ -0,0 +1,206 @@
+
+export type Staleness = 'fresh' | 'reconciling' | 'stale' | 'offline';
+
+export interface VaultWarning {
+  kind: string;
+  detail: string | null;
+}
+
+export type MountStatus =
+  | { state: 'opening' }
+  | { state: 'mounted'; path: string }
+  | { state: 'refused'; reason: string };
```

ESLint

[error] Replace `|·{·state:·'opening'·}⏎··|·{·state:·'mounted';·path:·string·}⏎·` with `{·state:·'opening'·}·|·{·state:·'mounted';·path:·string·}`

(prettier/prettier)